src/lib/y2storage/lvm_lv.rb

```diff
@@ -38,10 +38,21 @@ class LvmLv < BlkDevice
     #   @return [LvmVg] volume group the LV belongs to
     storage_forward :lvm_vg, as: "LvmVg"
 
-    # @!attribute stripes
+    # @!method lv_type
+    #   @return [LvType] type of the logical volume
+    storage_forward :lv_type, as: "LvType"
+
+    # @!method stripes
     #   Number of stripes. 0 if the LV is not striped
     #   @return [Integer]
     storage_forward :stripes
+
+    # @!method stripes=(num_stripes)
+    #   Set the number of stripes. The size of the LV must be a multiple of
+    #   the number of stripes and the stripe size. Thin LVs cannot be striped.
+    #
+    #   @param num_stripes [Integer]
+    #   @raise [Exception]
```

Comment:
I guess this is in fact a `Storage::Exception` and I guess it happens if the number is invalid according to the description above.